assets/app/scripts/app.js

```diff
@@ -338,6 +339,28 @@ angular
         return durationFilter(timestamp, null, omitSingle, precision) || existing;
       });
     }, 1000);
+  })
+  .run(function(extensionRegistry) {
+    extensionRegistry
```

Comment:
just add a comment to explain why we are using the extension registry for our own stuff so someone doesnt come in and question it later